azure-kusto-data/source/client.ts

```diff
@@ -43,6 +43,7 @@ export class KustoClient {
             "Accept": "application/json",
             "Accept-Encoding": "gzip,deflate",
             "x-ms-client-version": `Kusto.Node.Client:${pkg.version}`,
+            "Connection": "Keep-Alive",
```

Comment:
you might need to add an http/https agent too

https://github.com/axios/axios/issues/1846

Reply:
Thanks for pointing on that.